### PR TITLE
Add min and max replicas

### LINKED ISSUE #62: Expose min and max replicas on the HTTPScaledObject
### Use-Case

Users should be able to specify the minimum (0 or more) and maximum (1 or above) replicas that their app should have. These values map to the `ScaledObject` created behind the scenes.

### PR DESCRIPTION
**Pull request by khaosdoctor**
Adds new API to provide min and max replicas to scaledobjects

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] A PR is opened to update the documentation on [our docs repo](https://github.com/kedacore/keda-docs)

Fixes #62

### CODE REVIEW

File: operator/controllers/config/app_info.go [hunk 1]
```diff
@@ -6,6 +6,8 @@ type AppInfo struct {
 	Name                 string
 	Port                 int32
 	Image                string
+	MinReplicas          int32
```
Comment (arschles) [inline operator/controllers/config/app_info.go:9]: @khaosdoctor can we leave these out of the app config and use `httpso.Spec.MinReplicas`/`MaxReplicas` instead?

**Review by arschles — CHANGES_REQUESTED** [review]
looks good @khaosdoctor, one request only

File: examples/httpscaledobject.yaml [hunk 2]
```diff
@@ -6,3 +6,5 @@ spec:
     app_name: xkcd
     app_image: arschles/xkcd
     port: 8080
+    min_replicas: 5
+    max_replicas: 10
```
Comment (tomkerkhove) [inline examples/httpscaledobject.yaml:10]: How about this?
```suggestion
    replicas:
      min: 5
      max: 10
```

Comment (khaosdoctor) [reply examples/httpscaledobject.yaml:10]: I like it, will update the API to make this the default

Comment (khaosdoctor) [reply examples/httpscaledobject.yaml:10]: Resolved in last commit

File: operator/controllers/config/app_info.go (hunk 1 above)
Comment (khaosdoctor) [reply operator/controllers/config/app_info.go:9]: Also resolved in last commit

File: charts/keda-http-operator/crds/httpscaledobjects.http.keda.sh.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```
Comment (tomkerkhove) [inline charts/keda-http-operator/crds/httpscaledobjects.http.keda.sh.yaml:4]: We'll need to update this one

**Review by tomkerkhove — APPROVED** [review]

**Review by arschles — APPROVED** [review]
Looks great, thanks @khaosdoctor !

**Pull request merged**